authorization-common/src/main/java/com/sequenceiq/authorization/service/ResourceBasedCrnProvider.java

```diff
@@ -25,5 +27,9 @@ default String getResourceCrnByResourceName(String resourceName) {
         return Optional.empty();
     }
 
+    default Map<String, Optional<String>> getWithEnvironmentCrnsByResourceCrns(Collection<String> resourceCrns) {
```

Comment:
'with' not needed in the method name i guess

Reply:
thanks